cmake/dca_testing.cmake

```diff
@@ -105,6 +113,10 @@ function(dca_add_gtest name)
       target_compile_definitions(${name} PRIVATE DCA_HAVE_MAGMA)
     endif()
     cuda_add_cublas_to_target(${name})
+    if (DCA_ADD_GTEST_CUDA_MPI)
+      #We need to document which cuda aware openmpi requires this and which doesn't.
+      set(THIS_CVD_LAUNCHER "${CMAKE_SOURCE_DIR}/${CVD_LAUNCHER}")
```

Comment:
Is there a need to set a launcher specific for a test? Can't the same launcher work for all tests on a system? E.g., `--smpiargs='gpu'` will work for all tests and applications on summit. In general I think if the launch of the tests should be the same as the applications they are testing.

Reply:
As I have explained a number of times now --smpiargs or any other arg to the mpirun equivalent is not always sufficient to control the visibility/allocation of GPUs to ranks.

Reply:
and why can't this visibility script be used consistently for all the tests, or merged/fixed in a separate PR, as it has nothing to do with the ring G4 accumulation?

Reply:
It is needed for the test to be meaningful.